moveit_planners/ompl/ompl_interface/include/moveit/ompl_interface/model_based_planning_context.h

```diff
@@ -371,6 +371,9 @@ class ModelBasedPlanningContext : public planning_interface::PlanningContext
   void registerTerminationCondition(const ob::PlannerTerminationCondition& ptc);
   void unregisterTerminationCondition();
 
+  /** \brief Convert OMPL PlannerStatus to moveit_msgs::msg::MoveItErrorCode */
+  int32_t logPlannerStatus(og::SimpleSetupPtr ompl_simple_setup);
```

Comment:
The name and signature are somewhat misleading, aren't they? What about:
```suggestion
  int32_t errorCode(const ompl::base::PlannerStatus& status);
```

Reply:
yep, that's more useful in the case of `ompl_parallel_plan_` too  :+1: